### PR TITLE
Update PHP REST API post to use CLI for setup

### PR DESCRIPTION
**Pull request by mraible**
Updates [Build a Simple REST API in PHP](https://developer.okta.com/blog/2019/03/08/simple-rest-api-php) to use a streamlined setup.

### CODE REVIEW

**Comment by mraible** [comment]
@aaronpk @Endymion1977 are either of you able to update this post for the CLI? Since it's our most popular post, it'd be good to update it to use the latest SDK too, but that's not required.

**Comment by aaronpk** [comment]
Currently blocked on this issue:

https://github.com/okta/okta-jwt-verifier-php/issues/79

**Comment by mraible** [comment]
We might want to address this issue too. https://github.com/oktadeveloper/okta-php-core-rest-api-example/issues/5

**Comment by aaronpk** [comment]
Should be all set now!

File: _source/_posts/2019-03-08-simple-rest-api-php.md [hunk 1]
```diff
@@ -610,62 +614,46 @@ The client sends the access token to the REST API server;
...
 ```
 
 ## Add Authentication to Your PHP REST API
 
-We'll use the Okta JWT Verifier library. It requires a JWT library (we'll use `spomky-labs/jose`) and a PSR-7 compliant library (we'll use `guzzlehttp/psr7`). We'll install everything through composer:
+We'll use the Okta JWT Verifier library. It requires a JWT library (we'll use `firebase/php-jwt`) and a PSR-7 compliant library (we'll use `guzzlehttp/psr7`). We'll install everything through composer:
 
 ```bash
-composer require okta/jwt-verifier spomky-labs/jose guzzlehttp/psr7
+composer require okta/jwt-verifier firebase/php-jwt guzzlehttp/psr7
```
Comment (mraible) [inline _source/_posts/2019-03-08-simple-rest-api-php.md:656]: Is it possible to pin the versions for these libraries like you can with `npm install`?

Comment (aaronpk) [reply _source/_posts/2019-03-08-simple-rest-api-php.md]: Yep, just pushed a change for that.

**Comment by mraible** [comment]
If you're good with this, please approve @aaronpk.

**Review by aaronpk — APPROVED** [review]

**Pull request merged**